torch/_inductor/lowering.py

```diff
@@ -2915,6 +2917,7 @@ def index_put_impl_(self, indices, values, accumulate, check):
         indices_loaders,
         indexed_size,
         None,
+        add_asserts=check,
```

Comment:
Either change all the `check` parameters to `add_asserts` (a better name IMO) or name this one `check` as well.